### PR TITLE
Feature/components - Milestone 2

### PR DESCRIPTION
**Pull request by ClintonEnyinna**
The following where implemented: 

- [x] App component
- [x] Display component
- [x] Button component
- [x] ButtonPanel component

### CODE REVIEW

File: src/components/Display.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Display({ result }) {
+  return <div>{result}</div>;
+}
+
+export default Display;
```
Comment (shloch) [inline src/components/Display.jsx]: - enforce `result` to be a String with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]

- enforce `result` to have default valu of `'0'` with `defaultProps` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]

File: src/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Button({ name }) {
+  return <button type="button">{name}</button>;
+}
+
+export default Button;
```
Comment (shloch) [inline src/components/Button.jsx]: - enforce `name` to be string with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @ClintonEnyinna , 

Great job really...just some more validation to do

### [changes required]
check my comments below

**Review by fatymahmed — APPROVED** [review]
### Hi @ClintonEnyinna,
Great job making the requested changes 🎉🎉

- Please make sure you use gitflow for the other milestones in this project as mentioned on pathwright Screenshot from 2020-06-26 10-29-15

## status: Approved ✔️

**Pull request merged**